cdap-proto/src/main/java/co/cask/cdap/proto/Id.java

```diff
@@ -690,6 +690,10 @@ public String getId() {
         return id;
       }
 
+      public Flow getFlow() {
```

Comment:
It's no longer being used here, but I used it earlier and don't see a reason to remove.